src/test/java/tsundere/task/DeadlineTest.java

```diff
@@ -0,0 +1,40 @@
+package tsundere.task;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class DeadlineTest {
+    @Test
+    public void testDeadlineToString() {
```

Comment:
Consider naming your test case like this: `intDivision_nonZeroDivisor_success ` as specified in https://nus-cs2103-ay2324s2.github.io/website/schedule/week3/topics.html#c-to-java-junit-junit-basic in W3.7d C++ to Java → JUnit → JUnit: Basic